jcl/source/common/JclSysInfo.pas

```diff
@@ -2619,8 +2621,16 @@ function GetBIOSDate: TDateTime;
   RegSeparator: Char;
   {$ENDIF ~RTL150_UP}
 begin
-  if IsWinNT then
-    RegStr := RegReadString(HKEY_LOCAL_MACHINE, WinNT_REG_PATH, WinNT_REG_KEY)
+  if IsWinNT then begin
```

Comment:
The coding style guide mandates that `begin` be placed on its own line.
Also, if a branch of the `if`/`else` statement requires a `begin`/`end` block, then the other part must have it also.